Gopkg.toml

```diff
@@ -39,12 +39,16 @@
   version = "1.2.2"
 
 [[constraint]]
+  name = "gopkg.in/bblfsh/client-go.v3"
+  version = "3.0.0"
```

Comment:
3.1.0

Reply:
I forgot to mention I set this to the exact same version as gitbase, to have the same results from the queries and from the parsing we do in our side.
https://github.com/src-d/gitbase/blob/v0.18.0-beta.1/Gopkg.toml#L57

Reply:
But your lock file uses 3.1.0:
https://github.com/src-d/gitbase-web/pull/279/commits/540b3ecbe6ca02bb4d004795818b2d67bcc5867a#diff-bd247e83efc3c45ae9e8c47233249f18R316

v3.0.0 doesn't have SyntaxErr.

Reply:
That's right, I had to update to 3.1.0 for the error. It got updated in the lock file but not in this one. Done in 8f25ae5